Sources/Tracking/CustomerIO.swift

```diff
@@ -21,92 +21,3 @@ public extension CustomerIOInstance {
         screen(name: name, data: EmptyRequestBody())
     }
 }
-
-public extension CustomerIO {
-    /**
-     Initialize the shared `instance` of `CustomerIO`.
-     Call this function when the app launches, before using `CustomerIO.instance`.
-     */
-    @available(iOSApplicationExtension, unavailable)
-    static func initialize(
-        siteId: String,
-        apiKey: String,
-        region: Region,
-        configure configureHandler: ((inout SdkConfig) -> Void)?
-    ) {
-        var sdkConfig = SdkConfig.Factory.create(siteId: siteId, apiKey: apiKey, region: region)
-        let newDiGraph = DIGraph(sdkConfig: sdkConfig)
-
-        if let configureHandler = configureHandler {
-            configureHandler(&sdkConfig)
-        }
-
-        let implementation = DataPipeline.initialize(moduleConfig: DataPipelineConfigOptions.Factory.create(sdkConfig: sdkConfig))
-        initialize(implementation: implementation, diGraph: newDiGraph)
-    }
-
-    /**
-     Initialize the shared `instance` of `CustomerIO` for Notification Service Extension.
-     */
-    @available(iOS, unavailable)
-    @available(iOSApplicationExtension, introduced: 13.0)
-    static func initialize(
-        siteId: String,
-        apiKey: String,
-        region: Region,
-        configure configureHandler: ((inout NotificationServiceExtensionSdkConfig) -> Void)?
-    ) {
-        var newSdkConfig = NotificationServiceExtensionSdkConfig.Factory.create(siteId: siteId, apiKey: apiKey, region: region)
-        if let configureHandler = configureHandler {
-            configureHandler(&newSdkConfig)
-        }
-
-        let sdkConfig = newSdkConfig.toSdkConfig()
-        let newDiGraph = DIGraph(sdkConfig: sdkConfig)
-        let implementation = DataPipeline.initialize(moduleConfig: DataPipelineConfigOptions.Factory.create(sdkConfig: sdkConfig))
-
-        initializeSharedInstance(with: implementation, diGraph: newDiGraph)
```

Comment:
`initializeSharedInstance` can also be removed now, no? if so would be part of next PR?

Reply:
That's still being [used by `DataPipeline`](https://github.com/customerio/customerio-ios/blob/39e05f1cd02727d0d4ad67444f05c2c86e9ca31d/Sources/DataPipeline/CustomerIO.swift#L38), but yes there will be more changes in upcoming PRs, and it might either be removed or updated with new requirements.